### PR TITLE
[diffusion] Support LTX-2.5

### PR DESCRIPTION
**Pull request by AgainstEntropy**
## Modifications

This PR covered:

- Distilled and dev (full) ckpt
- Two-stage upscaler
- T2I, T2AV, I2AV
- TP
- SP
- CFG parallel
- Online FP8
- BCG
- DiT layerwise offload
- Cache-DiT
- Duration head (new)
- Diffusion video decoder (new)
  - tiling
- Cookbook

## Accuracy Tests

https://github.com/user-attachments/assets/0e6a2f81-70c0-42e5-ab2b-2700842eba3c

## Checklist

- [x] Format your code according to the [Format code with pre-commit](https://docs.sglang.io/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [x] Add unit tests according to the [Run and add unit tests](https://docs.sglang.io/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [x] Update documentation according to [Write documentations](https://docs.sglang.io/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.io/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.io/developer_guide/contribution_guide.html#benchmark-the-speed).
- [x] Follow the SGLang code style [guidance](https://docs.sglang.io/developer_guide/contribution_guide.html#code-style-guidance).

## Review and Merge Process

1. Ping Merge Oncalls to start the process. See the [PR Merge Process](https://github.com/sgl-project/sglang/blob/main/.github/MAINTAINER.md#pull-request-merge-process).
2. Get approvals from [CODEOWNERS](https://github.com/sgl-project/sglang/blob/main/.github/CODEOWNERS) and other reviewers.
3. Trigger CI tests with [comments](https://docs.sglang.io/developer_guide/contribution_guide.html#how-to-trigger-ci-tests) or contact authorized users to do so.
   - Common commands include `/tag-and-rerun-ci`, `/tag-run-ci-label`, `/rerun-failed-ci`
4. After green CI and required approvals, ask Merge Oncalls or people with Write permission to merge the PR.

---
### CI States

Latest PR Test (Base): :white_check_mark: [Run #31859829635](https://github.com/sgl-project/sglang/actions/runs/31859829635)
Latest PR Test (Extra): :x: [Run #31859829513](https://github.com/sgl-project/sglang/actions/runs/31859829513)

### CODE REVIEW

**Review by mickqian — COMMENTED** [review]
could you clean the document a bit?

**Review by mickqian — APPROVED** [review]

**Review by zijiexia — APPROVED** [review]

File: python/sglang/multimodal_gen/runtime/models/vaes/ltx_2_5_diffusion_decoder.py [hunk 1]
```diff
@@ -0,0 +1,1002 @@
...
+`kernel_size` positions. Like upstream this prefers NATTEN's fused `na3d`
+kernel and falls back to a FlexAttention block mask when NATTEN is missing, so
+NATTEN is an optimisation rather than a hard dependency.
+"""
+
+import math
+
+import torch
+import torch.nn.functional as F
+from diffusers.models.embeddings import PixArtAlphaCombinedTimestepSizeEmbeddings
```
Comment (mickqian) [inline python/sglang/multimodal_gen/runtime/models/vaes/ltx_2_5_diffusion_decoder.py]: is it possible that we make this vae fully native, e.g.:
1. use native implementation of `PixArtAlphaCombinedTimestepSizeEmbeddings`
2. adapt to native utilities, `ParallelTiledVAE` and  `wan_common_utils`?
3. is this vae able to parallel decode and tiled decode?

**Review by mickqian — COMMENTED** [review]
also, how does it work with cache-dit?

File: python/sglang/multimodal_gen/runtime/models/vaes/ltx_2_5_diffusion_decoder.py (hunk 1 above)
Comment (AgainstEntropy) [reply python/sglang/multimodal_gen/runtime/models/vaes/ltx_2_5_diffusion_decoder.py]: 1. cache-dit works (ltx2.5 reused existing DiT impl of ltx2 / 2.3
2. tiled decode of diffusion decoder was implemented here https://github.com/sgl-project/sglang/pull/34471/changes/2cd01c075d6db9e8b8759914ca8174d20414b6d7
3. working on native adaptation

Comment (AgainstEntropy) [reply python/sglang/multimodal_gen/runtime/models/vaes/ltx_2_5_diffusion_decoder.py]: > is this vae able to parallel decode and tiled decode?

Two parts:
1. It's actually not an VAE, but a small **diffusion** model especially for video decode
2. And yes I think it can do parallel decoding, will try to add this feature as a follow up PR

Comment (AgainstEntropy) [reply python/sglang/multimodal_gen/runtime/models/vaes/ltx_2_5_diffusion_decoder.py]: Follow up PR here: #36026
